src/main/scala/com/gd/twitterstreamingtToDatalake/StreamingTweetsJob.scala

```diff
@@ -0,0 +1,41 @@
+package com.gd.twitterstreamingtToDatalake
+
+import org.apache.log4j.Logger
+
+object StreamingTweetsJob extends SparkSessionWrapper {
+
+  //Set the Logger Level
+  val log = Logger.getLogger(StreamingTweetsJob.getClass.getName)
+
+  def main(args: Array[String]): Unit = {
+
+    if (validateConfiguration(args)) {
+      System.exit(1)
+    }
+    val arr = args.take(4)
+    TweetsIngestion.configureTwitter(arr)
```

Comment:
It's not that obvious what `arr` means.
Please read chapter #2 on meaningful names from "Clean Code" book. (reference in the reading list)

Reply:
Fixed with appropriate name of variable.